avm/res/container-service/managed-cluster/tests/e2e/azure/main.test.bicep

```diff
@@ -57,11 +61,14 @@ module diagnosticDependencies '../../../../../../../utilities/e2e-template-asset
   name: '${uniqueString(deployment().name, resourceLocation)}-diagnosticDependencies'
   params: {
     storageAccountName: 'dep${namePrefix}diasa${serviceShort}01'
-    logAnalyticsWorkspaceName: 'dep-${namePrefix}-law-${serviceShort}'
+    logAnalyticsWorkspaceName: 'dep-${namePrefix}-lawdiag-${serviceShort}-diag'
```

Comment:
why is the prefix “diag” used twice here? is there a particular reason for this?